checkstyle-rules.xml

```diff
@@ -50,6 +52,7 @@ Checkstyle configuration based on Sun's conventions, compliant with CAS coding c
     <module name="JavadocVariable">
       <property name="severity" value="error"/>
       <property name="scope" value="protected" />
+      <property name="ignoreNamePattern" value="log|logger|LOG|LOGGER"/>
```

Comment:
No comment required for loggers, yes. Properties can be ignored as long as they are conformant to Java beans. So for example, getStatus and setStatus do need docs, but getAuthenticationTokenBasedOnX() would need one. The property javadoc setting is controlled via `<property name="allowMissingPropertyJavadoc" value="true"/>`